cypress/e2e/dashboardsView.test.ts

```diff
@@ -731,4 +731,96 @@ describe('Dashboard', () => {
           })
       })
   })
+
+  // based on issue #18339
```

Comment:
I'm assuming the comment is for an issue in influxDB, but it might be beneficial to provide a direct link to the issue, or some context?